internal/worker_deployment_client.go

```diff
@@ -180,6 +180,8 @@ type (
 		ConflictToken []byte
 
 		// PreviousVersion - The Version that was current before executing this operation, if any.
+		// Deprecated, in favor of API idempotency. Use `Describe` before this API to get the previous
```

Comment:
nit: a blank line and a colon, for consistency with most other deprecated messages
```suggestion
        //
		// Deprecated: in favor of API idempotency. Use `Describe` before this API to get the previous
```